Source/Lib/Codec/EbSystemResourceManager.c

```diff
@@ -597,20 +604,24 @@ EB_ERRORTYPE EbReleaseObject(
 
     EbBlockOnMutex(objectPtr->systemResourcePtr->emptyQueue->lockoutMutex);
 
-    // Decrement liveCount
-    objectPtr->liveCount = (objectPtr->liveCount == 0) ? objectPtr->liveCount : objectPtr->liveCount - 1;
+    if (objectPtr->liveCount == EB_ObjectWrapperReleasedValue) {
+        SVT_LOG("Warning: %p is already released. Ignored the double EbReleaseObject() call \n", objectPtr);
+        return_error = EB_ErrorBadParameter;
+    } else {
+        // Decrement liveCount
+        objectPtr->liveCount = (objectPtr->liveCount == 0) ? objectPtr->liveCount : objectPtr->liveCount - 1;
 
-    if((objectPtr->releaseEnable == EB_TRUE) && (objectPtr->liveCount == 0)) {
+        if((objectPtr->releaseEnable == EB_TRUE) && (objectPtr->liveCount == 0)) {
```

Comment:
Please align with the format

Reply:
I surrounded those lines with an else { .. } line 611, so I increased the indent of those lines